packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1667,24 +1671,24 @@ class NotifierCheckbox extends StatelessWidget {
 
 class CheckboxSetting extends StatelessWidget {
   const CheckboxSetting({
-    Key key,
-    @required this.notifier,
-    @required this.title,
+    Key? key,
+    required this.notifier,
+    required this.title,
     this.description,
     this.tooltip,
     this.onChanged,
     this.enabled = true,
   }) : super(key: key);
 
-  final ValueListenable<bool> notifier;
+  final ValueListenable<bool?> notifier;
 
   final String title;
 
-  final String description;
+  final String? description;
 
-  final String tooltip;
+  final String? tooltip;
 
-  final void Function(bool newValue) onChanged;
+  final void Function(bool? newValue)? onChanged;
```

Comment:
these bools can be non-nullable

Reply:
nope: NotifierCheckbox.onChanged requires nullable parameter

Reply:
ack